chapter-05/final/money/presentation.go

```diff
@@ -0,0 +1,28 @@
+package money
```

Comment:
This file has no reason to be in this package. It is not `money`'s job to deal with strings, it is a presentation-layer problem.

Reply:
Do you think we should move the parsing or is it okay now?